### DISCUSSION TITLE
datetime strptieme problem

### DISCUSSION THREAD

**Discussion by fishjoe** [micropython/micropython#10532] [RP2040 / Pico] [answered]
```from datetime import datetime
timestr = "Tue, 17 Jan 2023 08:37:30 GMT"
dt_object = datetime.strptime(timestr, "%a, %d %b %Y %H:%M:%S %Z")
```

AttributeError: type object 'datetime' has no attribute 'strptime'

does anyone encountered this problem before? I tried also 
```import datetime```
then
```datatime.datetime.strptime()```
,  also not working

any help would be appreciated.

**Comment by peterhinch** [comment, marked as answer]
If you look at [the source](https://github.com/micropython/micropython-lib/blob/master/python-stdlib/datetime/datetime.py) it seems that `strptime` is not implemented. In many cases MicroPython libraries are subsets of the CPython ones.

**Reply by mattytrentini** [reply]
...but PRs are welcome! 😄
